### DISCUSSION TITLE
Unable to utilize TipTap Editor Properly

### DISCUSSION THREAD

**Discussion by fspplus** [awcodes/filament-tiptap-editor#438] [Q&A] [answered]
Hi I might need some help, 

Was not a fresh install of Filament and the plugin, suddenly got some error showing on consoles regarding a function of handleOpenModal in livewire.

<img width="1464" alt="Screenshot 2024-07-24 at 22 11 13" src="https://github.com/user-attachments/assets/2f61db9d-478f-402b-9ac7-90f1feca1282">

Now I can't seem to insert Images, Grids, and others to the editor.
Hope anyone can help me with this, quite new with Filament.

**Comment by awcodes** [comment, marked as answer]
try running `php artisan filament:assets`

**Reply by fspplus** [reply]
Thanks! it worked.
